tools/testbench/CMakeLists.txt

```diff
@@ -31,6 +33,12 @@ target_include_directories(testbench PRIVATE
 )
 endif()
 
+# Configuration time, make copy
+configure_file(${default_asoc_h} ${CMAKE_CURRENT_BINARY_DIR}/include/alsa/sound/asoc.h)
+
+# Build time
+target_include_directories(testbench SYSTEM PRIVATE "${CMAKE_CURRENT_BINARY_DIR}/include")
```

Comment:
off-line @singalsu said SYSTEM is not needed. Should be removed then.

Reply:
Yep, I just changed it back to PRIVATE.